projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -2153,7 +2153,10 @@ public function add_fediverse_creator_open_graph_tag( $tags ) {
 			// services can have multiple connections. Store them all in our array.
 			foreach ( $connections as $connection ) {
 				$connection_id   = $this->get_connection_id( $connection );
-				$mastodon_handle = $connection['external_display'] ?? '';
+				$connection_meta = $this->get_connection_meta( $connection );
+				$connection_data = $connection_meta['connection_data'];
+
+				$mastodon_handle = $connection_data['external_display'] ?? '';
```

Comment:
I'm afraid that won't work here. That information is not stored in `connection_data`. I'll update this PR on Monday to account for that.